model/src/pyrenew/model/rtinfectionsrenewal.py

```diff
@@ -85,32 +119,124 @@ def sample_rt(
         self,
         **kwargs,
     ) -> tuple:
+        """
+        Samples the Rt process
+
+        Parameters
+        ----------
+        **kwargs : dict, optional
+            Additional keyword arguments passed through to internal
+            sample_rt calls, should there be any.
+
+        Returns
+        -------
+        tuple
+
+        Notes
+        -----
+        TODO: More information in Returns.
+        """
         return self.Rt_process.sample(**kwargs)
 
     def sample_gen_int(
         self,
         **kwargs,
     ) -> tuple:
+        """
+        Samples the generation interval
+
+        Parameters
+        ----------
+        **kwargs : dict, optional
+            Additional keyword arguments passed through to internal
+            sample_gen_int calls, should there be any.
+
+        Returns
+        -------
+        tuple
+
+        Notes
+        -----
+        TODO: More information in Returns.
+        """
         return self.gen_int.sample(**kwargs)
 
     def sample_i0(
         self,
         **kwargs,
     ) -> tuple:
+        """
+        Samples the initial infections
+
+        Parameters
+        ----------
+        **kwargs : dict, optional
+            Additional keyword arguments passed through to internal
+            sample_i0 calls, should there be any.
+
+        Returns
+        -------
+        tuple
+
+        Notes
+        -----
+        TODO: More information in Returns.
+        """
         return self.i0.sample(**kwargs)
 
     def sample_infections_latent(
         self,
         **kwargs,
     ) -> tuple:
+        """
+        Samples the latent infections
+
+        Parameters
+        ----------
+        **kwargs : dict, optional
+            Additional keyword arguments passed through to internal
+            sample_infections_latent calls, should there be any.
+
+        Returns
+        -------
+        tuple
+
+        Notes
+        -----
+        TODO: More information in Returns.
+        """
         return self.latent_infections.sample(**kwargs)
 
     def sample_infections_obs(
         self,
         predicted: ArrayLike,
-        observed_infections: ArrayLike = None,
+        observed_infections: Optional[ArrayLike] = None,
         **kwargs,
     ) -> tuple:
+        """Sample number of hospitalizations
+
+        Parameters
+        ----------
+        predicted : ArrayLike
+            The predicted infecteds.
+        observed_hospitalizations : ArrayLike, optional
+            The estimates infections (to fit). Defaults to None.
+        **kwargs : dict, optional
+            Additional keyword arguments passed through to internal
+            sample_infections_obs calls, should there be any.
```

Comment:
The pass-through here is `sample()` calls generally, not specifically to calls to functions with the same name as this one.
```suggestion
            sample() calls, should there be any.
```